### ISSUE TITLE
Lack of error description [Settings Admin]

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1764]
When attempting to upload a new custom logo, the generic "Error" message is displayed without any indication of what went wrong. In this case, I discovered that the uploaded file was too large only after examining the console.
